src/index.js

```diff
@@ -451,6 +451,14 @@ function tab({shift = false, focusTrap = document} = {}) {
   }
 }
 
+function hover(element, init) {
```

Comment:
Since everything's going to eventually be async (#312), maybe we should make sure new utils are async and have a tick?

For now, I think this would be:

```javascript
await tick()
fireEvent...
await tick()
fireEvent..

..etc.
```

But eventually it'll just be:

```javascript
await fireEvent..
await fireEvent..
await fireEvent..
```

where `fireEvent` is our own wrapper around the dtlFireEvent:

```javascript
async function fireEvent(...args) {
  await tick()
  return fireEvent(...args)
}
```

Reply:
If you don't have a strong preference, I figured it would be easier to make that change separately.

Reply:
Right, all I'm saying is that as we're introducing something new, we should make it behave like it will in the near future (it should be async and there should be a tick between every fireEvent call).